.editorconfig

```diff
@@ -0,0 +1,15 @@
+# editorconfig for VS code
+
+root = true
```

Comment:
If we do not have to disable higher-level configurations, let's not disable them.

```suggestion
```